Add multi-view troubleshooting guide for sentry dart (#11308)
getsentry/sentry-docs: includes/troubleshooting/flutter/enable_multi_view.mdx

    }
    setState(() {
      _views = newViews;
    });
  }

  Widget _createViewWidget(FlutterView view) {
    return View(
      view: view,
      child: Builder(
        builder: widget.viewBuilder,
      ),
    );
  }

  @override
  void dispose() {
    WidgetsBinding.instance.removeObserver(this);
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return ViewCollection(views: _views.values.toList(growable: false));
  }
}
```

```js {filename:flutter_bootstrap.js}{tabTitle: web/flutter_bootstrap.js}
// flutter_bootstrap.js
{{ flutter_js; }}
{{ flutter_build_config; }}

_flutter.loader.load({
  onEntrypointLoaded: async function onEntrypointLoaded(engineInitializer) {
    let engine = await engineInitializer.initializeEngine({
      multiViewEnabled: true, // Enables embedded mode.
    });
    let app = await engine.runApp();
    // Make this `app` object available to your JS app.
    app.addView({ hostElement: document.querySelector("#left") });
    app.addView({ hostElement: document.querySelector("#right") });
  },
});
```

```html {filename:index.html}{tabTitle: web/index.html}
<!doctype html>
<html>
  <head>
    <!--

>           If you are serving your web app in a path other than the root, change the
>           href value below to reflect the base path you are serving from.
> 

Reviewer: ```suggestion
          If you're serving your web app in a path other than the root, change the
          href value below to reflect the base path you're serving from.

```

```dart {filename:main.dart}{tabTitle: lib/main.dart}
import 'dart:async';
import 'package:flutter/material.dart';
import 'package:sentry_flutter/sentry_flutter.dart';
import 'package:sentry_flutter/src/integrations/widgets_binding_integration.dart';
import 'dart:ui' show FlutterView;

> // ATTENTION: Change the DSN below with your own to see the events in Sentry. Get one at sentry.io
> const String exampleDsn =
>     'https://e85b375ffb9f43cf8bdf9787768149e0@o447951.ingest.sentry.io/5428562';

Reviewer 2: ```suggestion
const String exampleDsn = '___DSN___';
```

  onEntrypointLoaded: async function onEntrypointLoaded(engineInitializer) {
    let engine = await engineInitializer.initializeEngine({
      multiViewEnabled: true, // Enables embedded mode.
    });
    let app = await engine.runApp();
    // Make this `app` object available to your JS app.
    app.addView({ hostElement: document.querySelector("#left") });
    app.addView({ hostElement: document.querySelector("#right") });
  },
});
```

```html {filename:index.html}{tabTitle: index.html}
<!doctype html>
<html>
  ...
  <body>
    <div style="width: 100%; height: 100%; position: absolute;">
      <div id="left" style="width: 50%; height: 100%; float: left;"></div>
      <div id="right" style="width: 50%; height: 100%; float: left;"></div>
    </div>
    <!-- This script installs service_worker.js to provide PWA functionality to
       application. For more information, see:
       https://developers.google.com/web/fundamentals/primers/service-workers -->
    <script>
      if ("serviceWorker" in navigator) {
        window.addEventListener("load", function () {
          navigator.serviceWorker.register("flutter_service_worker.js");
        });
      }
    </script>

>     <script src="main.dart.js" type="application/javascript"></script>

Reviewer 3: `flutter_bootstrap.js` will add `main.dart.js` and register `flutter_service_worker`, so these two script tags could be removed

Author: thanks for the info.
